third_party/move/move-compiler-v2/src/pipeline/livevar_analysis_processor.rs

```diff
@@ -36,7 +36,7 @@ use std::{
 
 /// Annotation which is attached to function data.
 #[derive(Default, Clone)]
-pub struct LiveVarAnnotation(BTreeMap<CodeOffset, LiveVarInfoAtCodeOffset>);
+pub struct LiveVarAnnotation(pub BTreeMap<CodeOffset, LiveVarInfoAtCodeOffset>);
```

Comment:
Better if you can add a named method to get this, since `.0` is always pretty cryptic to see.

Reply:
I ended up changing the code to use the newtype pattern (https://doc.rust-lang.org/rust-by-example/generics/new_types.html) eliminating the use of `.0`.

Reply:
Yes, please lets avoid actually using named fields for singletons (and sometimes also for pairs and triples). The matching code becomes hard to read since the way how rustfmt works. Everybody seriously reading the code should easily remember what the single thing is. `let Wrapper(the_thing) = e` as deconstructor helps, but frankly `e.0` is also sometimes ok (and mentioned in the doc for newtype pattern).  It's kind of idiomatic, similar like `e.into()`